src/include/txn.h

```diff
@@ -255,17 +271,14 @@ struct __wt_txn {
 
     uint32_t forced_iso; /* Isolation is currently forced. */
 
-    /*
-     * Snapshot data:
-     *	ids >= snap_max are invisible,
-     *	ids < snap_min are visible,
-     *	everything else is visible unless it is in the snapshot.
-     */
-    uint64_t snap_min, snap_max;
-    uint64_t *snapshot;
-    uint32_t snapshot_count;
     uint32_t txn_logsync; /* Log sync configuration */
 
+    /* Snapshot data. */
+    WT_TXN_SNAPSHOT_DATA snapshot_data;
```

Comment:
suggest to remove _data in this as well as next.

Reply:
Fixed in 3523185a21d3fff8e71151f767b27376ee91d929